redash_toolbelt/examples/migrate.py

```diff
@@ -152,7 +152,7 @@ def get_all_users(client):
             "id": new_user["id"],
             "email": new_user["email"],
             "invite_link": PRESERVE_INVITE_LINKS and new_user["invite_link"] or "",
-            "disabled": user["is_disabled"],
```

Comment:
Line 143 is where the user is created. New users are enabled by default.

We use this line to keep track of which users should be disabled when the migration is complete. If we set this to `false` then we won't know which users to disable unless we hit the origin API again.